api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/context/MessageContextLoaderTest.java

```diff
@@ -18,28 +20,88 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.catchThrowableOfType;
 import static org.mockito.BDDMockito.given;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.lenient;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.*;
 import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleTestDataFactory.aValidPaymentCycle;
 
 @ExtendWith(MockitoExtension.class)
-class DetermineEntitlementMessageContextLoaderTest {
+class MessageContextLoaderTest {
 
+    public static final String CARD_ACCOUNT_ID = "myCardAccountId";
     @Mock
     private ClaimRepository claimRepository;
 
     @Mock
     private PaymentCycleRepository paymentCycleRepository;
 
     @InjectMocks
-    private DetermineEntitlementMessageContextLoader loader;
+    private MessageContextLoader loader;
 
     //These calls need to be lenient as Mockito sees mocking the same method call multiple times as a sign of an error
     //and will fail the test without setting the mode to lenient.
     @Test
-    void shouldSuccessfullyLoadContext() {
+    void shouldSuccessfullyLoadPaymentContext() {
+        //Given
+        Claim claim = ClaimTestDataFactory.aValidClaim();
+        UUID claimId = claim.getId();
+        PaymentCycle paymentCycle = aValidPaymentCycle();
+        UUID paymentCycleId = paymentCycle.getId();
+        given(claimRepository.findById(any())).willReturn(Optional.of(claim));
+        when(paymentCycleRepository.findById(paymentCycleId)).thenReturn(Optional.of(paymentCycle));
```

Comment:
Any reason why you're using `when` here rather than `given`? Same in test below too.